CoEpi/ui/HealthQuiz/HealthQuizViewModel.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  HealthQuizViewModel.swift
+//  CoEpi
+//
+//  Created by Johnson Hsieh on 3/26/20.
+//  Copyright © 2020 org.coepi. All rights reserved.
+//
+
+class HealthQuizViewModel {
+    weak var delegate: HealthQuizViewModelDelegate?
+
+    private var questions: Array<Question> = [
+        Question(id: 1, text: "Fever", checked: false),
+        Question(id: 2, text: "Tiredness", checked: false),
+        Question(id: 3, text: "Dry cough", checked: false),
+        Question(id: 4, text: "Muscle aches", checked: false),
+        Question(id: 5, text: "Nasal aongestion", checked: false),
+        Question(id: 6, text: "Runny nose", checked: false),
+        Question(id: 7, text: "Sore throat", checked: false),
+        Question(id: 8, text: "Diarrhea", checked: false),
+        Question(id: 9, text: "Difficulty breathing", checked: false),
+        Question(id: 10, text: "Loss of smell/taste", checked: false),
+        Question(id: 11, text: "None of the Above", checked: false)
+    ]
+    
+    var numQuestions: Int { return questions.count }
+
+    func question(at index: Int) -> Question {
+        return questions[index]
+    }
+
+    func handleAnswer(question: Question, checked: Bool) {
+        // TODO: represent as a map
+        let index: Int? = questions.firstIndex { $0.id == question.id }
+        guard let i = index else { return }
+
+        let q = questions[i]
+        questions[i] = Question(id: q.id, text: q.text, checked: checked)
```

Comment:
Maybe write an extension to replace an item in an array?

Reply:
Added a new init instead - `init(oldQuestion: Question, newCheckedValue: Bool)`